src/test/java/ir/sahab/nexus/plugin/tag/IntegrationTest.java

```diff
@@ -268,6 +272,7 @@ private void assertDefinitionEquals(TagDefinition expected, Tag actual) {
 
     @AfterClass
     public static void tearDown() {
+        repositoryRule.after();
```

Comment:
Why not define RepositoryRule as a ClassRule??

Reply:
It's already a rule, I was testing sth and I forgot to remove it.